src/components/Layout/Sidebar/Sidebar.tsx

```diff
@@ -106,6 +110,19 @@ export function Sidebar({ isExpanded, onCollapseSidebar }: SidebarProps) {
                 label="Data files"
                 files={dataFiles}
                 noFilesMessage="No data files found"
+                actions={
+                  <Tooltip content="Import data file" side="right">
+                    <IconButton
+                      asChild
+                      aria-label="Import data file"
+                      variant="ghost"
+                      size="1"
+                      onClick={handleImportDataFile}
+                    >
+                      <PlusIcon />
```

Comment:
I don't really like the plus icon here, because it's used to create resources in other places (recorder/generator file trees). If you a radix icon that works better, please suggest it

Reply:
Maybe `download` icon? 
![screencapture 2025-01-24 at 11 03 02](https://github.com/user-attachments/assets/0a3b985a-8583-4886-a395-d3c6c9e34677)

If you do a google search for "import icon" most of them consist of an arrow pointed into a rectangle

Reply:
Maybe the `upload` one then?
<img width="46" alt="image" src="https://github.com/user-attachments/assets/673b1aa6-3f57-46be-8ca3-97f6bf1eb2a1" />

Reply:
This feels more like "share" action to me, but I don't have strong opinion 👍

Reply:
what about the `file plus` ?
<img width="28" alt="image" src="https://github.com/user-attachments/assets/3336af21-163e-4dbf-9b8c-0cf612975428" />

Reply:
I think I like this suggestion the most